src/commands/commerce/extension/unmap.ts

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { flags, SfdxCommand } from '@salesforce/command';
+import { Messages, Org, SfdxError } from '@salesforce/core';
+import { QueryResult } from '@mshanemc/plugin-helpers/dist/typeDefs';
+import { forceDataSoql, forceDataRecordDelete } from '../../../lib/utils/sfdx/forceDataSoql';
+import { StatusFileManager } from '../../../lib/utils/statusFileManager';
+import { Result } from '../../../lib/utils/jsonUtils';
+
+Messages.importMessagesDirectory(__dirname);
+
+const TOPIC = 'extension';
+const CMD = `commerce:${TOPIC}:unmap`;
+const msgs = Messages.loadMessages('@salesforce/commerce', 'store');
```

Comment:
It might be worthwhile to pull messages out of `store` and put it in its own separate JSON file like `extension.json` within the messages directory, just to keep the message pattern consistent throughout the project